junit-platform-console/src/main/java/org/junit/platform/console/options/AvailableOptions.java

```diff
@@ -59,8 +59,15 @@
 		disableAnsiColors = parser.accepts("disable-ansi-colors",
 			"Disable ANSI colors in output (not supported by all terminals).");
 
-		hideDetails = parser.accepts("hide-details",
-			"Hide details while tests are being executed. Only show the summary and test failures.");
+		// TODO Remove "hide-details" before final release. Only here to make "legacy" Gradle plugin happy.
+		// If removed, an old Gradle:junitPlatformTest will report:
+		//   Exception in thread "main" joptsimple.UnrecognizedOptionException: hide-details is not a recognized option
+		parser.accepts("hide-details", "@Deprecated. Use '--details=hidden' instead.");
```

Comment:
Could stay for good, and be interpreted as `--details=hidden`. Overriding any setting made with `-details` or if both are specified by the user, print a warning saying _Its enough to pass `-details=hidden`._ or even exit with an error message.